packages/host/app/components/operator-mode/code-submode/format-chooser.gts

```diff
@@ -68,6 +69,7 @@ export default class FormatChooser extends Component<Signature> {
         margin-right: var(--boxel-sp-xxs);
         border: 1px solid
           var(--boxel-format-chooser-border-color, var(--boxel-700));
+        box-shadow: var(--boxel-deep-box-shadow);
```

Comment:
This updates is not related to this PR but this is the small fix for format chooser button.